pytensor/tensor/rewriting/ofg.py

```diff
@@ -0,0 +1,41 @@
+from pytensor.compile.builders import OpFromGraph
+from pytensor.graph import node_rewriter
+from pytensor.tensor.rewriting.basic import register_specialize
+
+
+@register_specialize("opfromgraph")
+@node_rewriter([OpFromGraph])
```

Comment:
```suggestion
@node_rewriter([AllocDiag])
```

Reply:
I thought we want to inline everything?

Reply:
No why would we? Some OFG we want to inline others not. This would steal users ability to set inline=False

Reply:
Well we definitely want to do all these "wrapper" OFGs, like `kron` and `einsum` yes?

Reply:
Sure you just add them to the tracks as we implement them. It can have more than one entry

Reply:
Einsum will probably be treated differently though due to the static vs non static shape case